install/connections.csv

```diff
@@ -221,9 +221,9 @@ GISX,10,Patutahi,Portable,,2006-07-22T21:00:01Z,2006-08-17T05:00:00Z
 GLAA,10,Gillespies Beach,Short Period,,1997-09-20T07:06:00Z,1997-10-28T02:47:00Z
 GLET,10,Glencoe,Short Period,,1993-04-12T09:46:00Z,1993-04-25T09:39:00Z
 GLKZ,10,Raoul Island Green Lake,,,2009-05-19T00:00:00Z,9999-01-01T00:00:00Z
-GLKZ,40,Raoul Island Green Lake,,,2009-05-19T00:00:00Z,9999-01-01T00:00:00Z
-GLKZ,80,Raoul Island Green Lake,,,2009-05-18T00:00:01Z,9999-01-01T00:00:00Z
-GLKZ,81,Raoul Island Green Lake,,,2009-05-18T00:00:01Z,9999-01-01T00:00:00Z
+GLKZ,40,Raoul Island Green Lake,,,2009-05-19T00:00:00Z,2023-01-30T20:17:32Z
```

Comment:
for the sake of the future Delta upgrade, may worth to add the pin numbers ;)

Reply:
mmh good point. do we know what they are or how I can obtain the info?

Reply:
We m ight get some hints  there @elidana  ;)
https://github.com/GeoNet/tickets/issues/12320#issuecomment-1407912192

(perhaps 40 is 3, 80 is 4,  81 is 5  - fropm 0 to 5 pins  )

Although this isn t mandatory  - more like an exercise - sorry to bother everyone with my pins today!

Reply:
channels 40, 80, and 81 are on pin 3 corresponding to channels 4,5, and 6 of the q330

the three seismic channels (10 HHZ | N | E) are on pin 0 corresponding to channels 1, 2, and 3 (these are most definitely staying 🤣 )

cc @salichon @elidana

Reply:
thanks @CBurton90!! will add these then!

Reply:
oh so....would be `80` on pin number `3` and `81` on pin number 4?

Reply:
correction
40 = pin n.3
80 = pin n.4
81 = pin n. 5
?

Reply:
That's correct!

Reply:
awesome, thanks for checking!